### PR TITLE
Feature/quiz automated timeouts

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

File: src/TransformCore/PHE/ExporterBundle/Command/CSVExporterCommand.php [hunk 1]
```diff
@@ -61,21 +56,18 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $offsetMinutes = ($input->getArgument('offset')) ?: 240;
+        parent::execute($input, $output);
+
+        $offsetMinutes = ($input->getOption('offset')) ?: 240;
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/ExporterBundle/Command/CSVExporterCommand.php]: Default to `240`? Would it better to do it as a `const`?

Comment (brettminnie) [inline src/TransformCore/PHE/ExporterBundle/Command/CSVExporterCommand.php]: Good call, has also changed, I'll update

File: src/TransformCore/PHE/ExporterBundle/Command/CSVExporterCommand.php [hunk 2]
```diff
@@ -112,22 +98,21 @@ protected function encryptFileForTransfer($filePath)
 
             $service->encryptFile($filePath);
 
-            $this->getLogger()->info("$filePath encrypted successfully");
+            $this->writeOutput("$filePath encrypted successfully");
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/ExporterBundle/Command/CSVExporterCommand.php]: You are mixing styles...

`"$filePath encrypted successfully"` and `$filePath . ' encrypted successfully'`

File: src/TransformCore/PHE/HayApi/PersistenceBundle/Command/HayCommand.php [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+    /**
+     * @var OutputInterface
+     */
+    protected $output;
+
+    /**
+     * @var bool
+     */
+    protected $squelch = false;
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Command/HayCommand.php]: `$isSquelch`?

Comment (brettminnie) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Command/HayCommand.php]: isSquelched would be correct for the accessor, not using it as it's only internal

Comment (eddiejaoude) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Command/HayCommand.php]: k

Comment (eddiejaoude) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Command/HayCommand.php]: :+1:

**Pull request merged**